eve/main.yml

```diff
@@ -127,7 +127,7 @@ stages:
     - ShellCommand:
         name: 'Build doc'
         haltOnFailure: true
-        command: tox --workdir /tmp/tox -e docs -- html latexpdf
+        command: tox --workdir /tmp/tox -e docs -- html # latexpdf
```

Comment:
Why are we disabling latexpdf in the CI? Is there some errors from the pdf generation?